_data/README.md

```diff
@@ -0,0 +1,61 @@
+# data.csv
+
```

Comment:
```suggestion
```suggestion
`data.csv` file should not be edited by hand. It is generated automatically by the workshop coordinator when all the information is available.
```
```